docker-compose/run-test.sh

```diff
@@ -1,50 +0,0 @@
-#!/bin/sh
```

Comment:
The `script` field in `travis.yml` should be updated to `docker-compose/scripts/run-test.sh`